c7n/resources/ebs.py

```diff
@@ -252,6 +254,41 @@ def process(self, resources, event=None):
         return self.get_matching_aliases(resources)
 
 
+@filters.register('fault-tolerant')
+class FaultTolerantSnapshots(Filter):
```

Comment:
we want to have doc strings on all filters with sample usage. ie. it would be good to explain what fault tolerant means here and an example usage.